docs/book/src/appendix/keywords.md

```diff
@@ -0,0 +1,89 @@
+# [Appendix A: Keywords](#appendix-a-keywords)
+
+The following list contains keywords that are reserved for current or
+future use by the Sway language. As such, they cannot be used as
+identifiers. Identifiers are names of functions, variables,
+parameters, modules, constants, attributes, types or
+traits, ect.
+
+## [Keywords Currently in Use](#keywords-currently-in-use)
+
+The following is a list of keywords currently in use, with their
+functionality described.
+
+- `as` - perform primitive casting, or rename items in `use` statements
```

Comment:
I'm not as up-to-date on new language features so I haven't heard of `as` being in `Sway`. Can you point me at an example which uses this?

If there isn't one then this should be removed.

Reply:
Pretty sure we at least have 
```sway
use type::a as alias_name
```

Reply:
> Pretty sure we at least have
> 
> ```sway
> use type::a as alias_name
> ```

Just tested and this is the only way it can be used. I went ahead and kept it listed but gave an example of its only use.

Reply:
This code compiles:
```sway
script;

use std::logging::log as logger; // alias here

struct U128 {
    a: u64,
    b: u64,
}

fn main() {
    let x = U128 { a: 0, b: 5 };
    logger(x);
}

```